bin/job-presync.sh

```diff
@@ -10,7 +10,7 @@ release=$1
 run_policy=${2:-'OnSpecChange'}
 
 is_deployed=false
-hf list | grep $release >/dev/null && is_deployed=true
```

Comment:
I recommend to use 
```
set -euo pipefail
```
at the beginning of the script

Reply:
those are defaults that we don't want in this case

Reply:
I disagree. The script should fail as soon as possible. Otherwise an error from presync will be populated somewhere else and create a noise, which makes it harder to find a root cause.